gpt_bpe_test.go

```diff
@@ -1193,141 +1532,85 @@ func TestModelDownloadLlama(t *testing.T) {
 
 	// Check for tokenizer.model
 	tokenizerConfigPath := destPath + "/tokenizer.model"
-	if _, err := os.Stat(tokenizerConfigPath); err == nil {
-		fmt.Println("tokenizer.model exists")
-
-	} else if errors.Is(err, os.ErrNotExist) {
-		t.Errorf("tokenizer.model does not exist. %s", err)
-
-	} else {
-		t.Errorf("Error checking for tokenizer.model. %s", err)
-	}
+	assertFileExists(t, tokenizerConfigPath)
```

Comment:
I like how keeping this separate provides readability but this can be condensed into a for loop